recipe/meta.yaml

```diff
@@ -1,50 +1,64 @@
-{% set version = "0.21.2" %}
-
+{% set name = "woodwork" %}
+{% set version = "0.25.1" %}
 
 package:
-  name: woodwork
+  name: {{ name }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/w/woodwork/woodwork-{{ version }}.tar.gz
-  sha256: 01eb8c7fc01240bea2329d45ee5a2b2a016c7e060199e82b5aac262efad183b8
+  url: https://pypi.io/packages/source/{{name[0]}}/{{name}}/{{name}}-{{version}}.tar.gz
+  sha256: 73cb38e09229216cb7cbbaf2e4352b34d6e40ac44bf774c17e94a9047c7e98e3
+  patches:                        # [win]
+    - remove-absolute-path.patch  # [win]
 
 build:
   entry_points:
     - woodwork = woodwork.__main__:cli
   number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . --no-deps -vv
+  skip: True  # [py<38]
+  script: {{ PYTHON }} -m pip install . --no-deps --no-build-isolation -vv
 
 requirements:
+  build:                          # [win]
+    - m2-patch                    # [win]
   host:
     - pip
-    - python >=3.8.*
+    - python
+    - setuptools >=61.0.0
+    - wheel
   run:
-    - importlib-resources >=5.10.0
-    - numpy >=1.21.0
-    - pandas !=1.4.2, >=1.4.0
-    - python >=3.8.*
+    - importlib_resources >=5.10.0
+    - numpy >=1.22.0,<1.25.0
+    - pandas >=1.4.3
+    - python
     - python-dateutil >=2.8.1
     - scikit-learn >=0.22
-    - scipy >=1.4.0
+    - scipy >=1.10.0
+
 test:
   imports:
     - woodwork
     - woodwork.tests
   requires:
+    - pip
+# Skip tests on s390x; missing moto and pyarrow
+{% if not s390x %}
     - boto3 >=1.10.45
     - moto >=3.0.7
     - pyarrow >=4.0.1
-    - pytest ==7.0.1
-    - pytest-cov ==2.10.1
-    - pytest-xdist ==2.1.0
+    - pytest >=7.0.1
+    - pytest-cov >=2.10.1
+    - pytest-xdist >=2.1.0
     - smart_open >=5.0.0
   source_files:
     - woodwork/tests/*
+{% endif %}
   commands:
-    - pytest woodwork/tests
-    - pip check
+    - pytest woodwork/tests  # [not (s390x or win)]
+    # Hard coded paths cause a few tests to break on Windows.
+    - pytest woodwork/tests --ignore "woodwork/tests/accessor/test_serialization.py" --ignore "woodwork/tests/utils/test_read_file.py"  # [win]
```

Comment:
Which tests were failing? Looking at the source, it was not obvious to me at least.

You can also use `-k (not test_read_file)` to exclude tests based on patterns, and --verbose to get more details on test failures in the logs.

Reply:
test_read_file and test_to_disk. In both cases, it was throwing a fatal error and ending the entire test session prematurely, so there may be other tests in those files that fail.

Reply:
is it something to do with the `tmpdir` argument for [test_to_disk](https://github.com/alteryx/woodwork/blob/1bf1cdceee0ade1f6d8411ebffe17776b7f071e9/woodwork/tests/accessor/test_serialization.py#L444C1-L444C50)? where do the sample_df and tmpdir arguments to these tests come from?

Reply:
just wondering where these hardcoded file paths come from and if it's easily fixable

Reply:
@danpetry Looks like it's using the `tempdir` [fixture in Pytest](https://docs.pytest.org/en/6.2.x/tmpdir.html#tmpdir-factory-example), which creates a directory in the system's temp folder, typically /tmp/pytest-NUM for Unix based systems. I can't figure out where it usually puts it for Windows, but I'm guessing it doesn't have access to that location. That said, I can override it in the pytest call, so I'm going to try that.

Reply:
No luck with that. Looks like `pytest-xdist` doesn't respect the `--basetemp` flag. Looks like it's been brought up as an issue a few times with no resolution.

Reply:
Thanks for the thorough investigation